colcon_mixin/mixin/__init__.py

```diff
@@ -9,23 +9,41 @@
 from colcon_core.logging import colcon_logger
 import yaml
 
+from colcon_core.environment_variable import EnvironmentVariable
```

Comment:
Move this up to the top of the file with the other imports, and move `COLCON_MIXIN_PATH` down to above `get_mixin_path`?